content/20.treatments.md

```diff
@@ -286,7 +286,20 @@ When these identifiers are followed back to the registry, most trials claim they
+The trial studied efficacy of HCQ in conjunction with standard of care (SOC) vs only SOC in 150 infected patients assigned randomly to two respective groups (75 each group). 
+Most of the cases (98%) in this cohort were mild to moderate with average age of patients being 46 years.
+The primary end point of the study was the negative conversion rate of SARS-CoV-2 in 28 days and the investigators found no difference in this parameter between the groups. 
+The secondary end points were amelioration of the symptoms of the disease. 
+The median time of symptoms alleviation in both cases was similar (19 days in HCQ+SOC vs 21 days in SOC). 
+However, an interesting aspect of the analyses by the investigators showed that, HCQ was efficacious in alleviation of symptoms in patients receiving no antiviral treatment in post hoc analysis. 
+Additionally, there was an increase in lymphocytes in SOC+HCQ group as opposed to only SOC group at a borderline p value (p=0.57). Following this, the authors hypothesized that addition of HCQ to the current SOC could decrease inflammatory response and subsequently prevent multiorgan failure and death. 
+Additionally, one of the key results of this trial was that the 30% of the patients that received SOC+ HCQ had adverse outcomes as opposed to 8.8% in only SOC. The most adverse outcomes being diarrhea in SOC+HCQ group(10%). 
+The study has few limitations. One of the limitations of the study was that the cohort mostly consisted of patients with mild/moderate disease symptoms with an average age group 46 and did not include aged group known to be at high risk category of the disease. 
+Although, authors claimed that HCQ could have beneficial effect hindering disease progression, multiorgan failure and death in COVID-19, that is yet to be tested for severe cases.
+Indeed, larger population needs to be tested to validate the increase of lymphocytes in SOC+HCQ as although, the result was promising, it lacked statistical significance.  Additionally, SOC used in this study included the use of antivirals which could provide confounding effects. Thus, to better understand the drug effects of HCQ, a better study design needs to be outlined, where SOC should not involve the use of anti-virals. 
+In this trial, the samples collected from the upper respiratory tract introduced false negatives and thus finding biomarker for the disease is incumbent. Future studies to unravel serum/ urinary/ Bronchoalveolar lavage (BAL) markers for the disease is  warranted.
+Another limitation of the study was that the HCQ treatment was provided at 16 day delay from the symptom onset.  A quicker treatment regimen with HCQ needs to be investigated for early disease management. Overall, the study holds promising data albeit needs validation in independent population cohort. Also from safety point of view, the adverse effect in manageable.
 ##### Summary
 
 In vitro evidence shows HCQ may be an effective therapeutic against SARS-CoV-2 and COVID-19. 
```

Comment:
_In vitro_ evidence shows Choloroquine and Remdesivir may be an effective therapeutic against SARS-CoV-2  (doi:10.1038/s41422-020-0282-0). Using African green monkey kidney cells and human liver cancer cells, the study evaluated the antiviral efficiency of 5 FDA-approved (ribavirin, penciclovir, nitazoxanide, nafamostat, choloroquine, remdesivir, and favipiravir)  drugs against a clinical isolate of 2019-nCoV _in vitro_. The authors quantified viral copy numbers in cell supernatant and confirmed the results using viral nucleoprotein expression through immunofluorescence. Both Choloroquine and Remdesivir blocked virus infection at low-micromolar concentration and showed a high selectivity index (SI). This study investigated basic mechanisms using African green monkey kidney cells and human liver cancer cells. Including a lung cell type might have been more applicable to use a lung cell type, since we know that the lungs are what is affected by SARS-CoV-2. Further studies in vivo are required to study the effects of Choloroquine and Remdesivir.

Reply:
Looks great.